Tests/DatadogTests/Datadog/CrashReporting/CrashReporterTests.swift

```diff
@@ -25,7 +25,7 @@ class CrashReporterTests: XCTestCase {
         let crashReporter = CrashReporter(
```

Comment:
This is part of:
```swift
testGivenPendingCrashReport_whenLoggingOrRUMIntegrationIsEnabled_itSendsAndPurgesTheCrashReport
```
test and I think it doesn't make sense anymore, because the `integration` is always there.

What still makes sense however, is IMO asserting that crash report is purged after being sent to `integration`, so renaming this test to something like:
```swift
testWhenPendingCrashReportIsFound_itIsSentToIntegrationAndPurged
```